README.md

```diff
@@ -424,12 +417,20 @@ concept concepts::meta =
 
 ```cpp
 /**
- * Variable template to represents single type
+ * Variable template whichi represents single type
```

Comment:
Looks like a small typo `whichi` (unless this is a word I'm unfamiliar with).

Reply:
Fixed, thanks